fog/distribution/src/main.rs

```diff
@@ -409,7 +409,8 @@ fn select_spendable_tx_outs(
                     get_tx_out_shared_secret(account.view_private_key(), &public_key);
 
                 let (amount, _blinding_factor) = tx_out
-                    .masked_amount
+                    .get_masked_amount()
+                    .unwrap_or_else(|err| panic!("TX missing masked value: {}", err))
```

Comment:
Nit: I think you can replace this line with `expect("Tx missing masked value)` because `expect` will  append `: {ERROR MESSAGE}`. (https://doc.rust-lang.org/std/result/enum.Result.html#method.expect)